contest/templates/contest/contest.html

```diff
@@ -52,19 +52,20 @@ <h1 class="panel-title" style="text-align:center;">Time</h1>
         <h1 class="panel-title" style="text-align:center;">Clarification</h1>
       </div>
       <div id="contest_clarification">
-        <table class="table">
+        <table class="table" style="table-layout: fixed; width: 100%">
           <thead>
             <tr>
               {% if user|can_reply:contest %}
-                <th></th>
+                <th width="5%"></th>
               {% endif %}
-              <th>#</th>
-              <th>Problem</th>
-              <th>Asker</th>
-              <th>Description</th>
-              <th>Reply</th>
-              <th>Replier</th>
-              <th>Reply Time</th>
+              <th width="2%">#</th>
+              <th width="15%">Problem</th>
+              <th width="10%">Asker</th>
+              <th width="20%">Description</th>
+              <th width="20%">Reply</th>
+              <th width="10%">Replier</th>
+              <th width="10%">Reply Time</th>
+              <th width="5%">For all team</th>
```

Comment:
Maye it would cause some problem like... 
![2015-04-29 17 23 51](https://cloud.githubusercontent.com/assets/9431996/7388020/9a1f1b80-ee94-11e4-8d7e-763ac4f4e9d6.png)